<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-01-30T10:29:06 IST" name="Default suite" finished-at="2025-01-30T10:29:30 IST" duration-ms="23973">
    <groups>
    </groups>
    <test started-at="2025-01-30T10:29:06 IST" name="Default test" finished-at="2025-01-30T10:29:30 IST" duration-ms="23973">
      <class name="Core.SecurityManagement.CallCentre_UserManagement">
        <test-method is-config="true" signature="reference()[pri:0, instance:Core.SecurityManagement.CallCentre_UserManagement@75db5df9]" started-at="2025-01-30T10:28:02 IST" name="reference" finished-at="2025-01-30T10:29:06 IST" duration-ms="63615" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reference -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:Core.SecurityManagement.CallCentre_UserManagement@75db5df9]" started-at="2025-01-30T10:29:06 IST" name="ExtentTestManagerStartTest" finished-at="2025-01-30T10:29:06 IST" duration-ms="655" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void Core.SecurityManagement.CallCentre_UserManagement.Login_to_Beacon_FCM() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Login_to_Beacon_FCM()[pri:1, instance:Core.SecurityManagement.CallCentre_UserManagement@75db5df9]" started-at="2025-01-30T10:29:06 IST" name="Login_to_Beacon_FCM" finished-at="2025-01-30T10:29:07 IST" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login_to_Beacon_FCM -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:Core.SecurityManagement.CallCentre_UserManagement@75db5df9]" started-at="2025-01-30T10:29:07 IST" name="ExtentTestManagerStartTest" finished-at="2025-01-30T10:29:07 IST" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void Core.SecurityManagement.CallCentre_UserManagement.Access_User_Management_Page() throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method signature="Access_User_Management_Page()[pri:2, instance:Core.SecurityManagement.CallCentre_UserManagement@75db5df9]" started-at="2025-01-30T10:29:07 IST" name="Access_User_Management_Page" finished-at="2025-01-30T10:29:15 IST" duration-ms="8429" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Access_User_Management_Page -->
        <test-method signature="ExecutiveID_ECP_Validations(java.util.Map)[pri:13, instance:Core.SecurityManagement.CallCentre_UserManagement@75db5df9]" started-at="2025-01-30T10:29:17 IST" name="ExecutiveID_ECP_Validations" data-provider="TestData" finished-at="2025-01-30T10:29:28 IST" duration-ms="11001" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{AlphabeticInputName=, AlphaNumericInputName=, Branch=, Email=caldlcyuimytu@gmail.com, EmailWithoutDomain=, ExecutiveID=2332, HeadOffice=, InvalidEmailId=, InvalidUserName=, Name=Agent Six, NumberEightDigits=, NumberWithLetters=, NumericInputName=, OrganizationType=, OrganizationType2=, OrganizationType3=, OrganizationType4=, Phone=9876756533, PwdOriginal=, Region=, Run=Yes, Testrole=, TestScenario=CallCentre_UserManagment, UNOriginal=, UserManagementPageRole=, UsernameInUNPage=, ZoneCo=}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.findElement(org.openqa.selenium.By)" because "this.driver" is null
at Core.SecurityManagement.UserManagement_Page.ECPValidationAlphabets(UserManagement_Page.java:394)
at Core.SecurityManagement.CallCentre_UserManagement.ExecutiveID_ECP_Validations(CallCentre_UserManagement.java:334)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExecutiveID_ECP_Validations -->
        <test-method is-config="true" signature="ExtentTestManagerStartTest(java.lang.reflect.Method)[pri:0, instance:Core.SecurityManagement.CallCentre_UserManagement@75db5df9]" started-at="2025-01-30T10:29:17 IST" name="ExtentTestManagerStartTest" finished-at="2025-01-30T10:29:17 IST" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void Core.SecurityManagement.CallCentre_UserManagement.ExecutiveID_ECP_Validations(java.util.Map) throws java.lang.Throwable]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ExtentTestManagerStartTest -->
        <test-method is-config="true" signature="AfterClass()[pri:0, instance:Core.SecurityManagement.CallCentre_UserManagement@75db5df9]" started-at="2025-01-30T10:29:28 IST" name="AfterClass" finished-at="2025-01-30T10:29:30 IST" duration-ms="1584" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterClass -->
      </class> <!-- Core.SecurityManagement.CallCentre_UserManagement -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
